src/actions/uploadActions.ts

```diff
@@ -0,0 +1,13 @@
+export const uploadFiles = () => {
+  return {
+    type: "uploaded",
+    filesUploaded: true,
+  };
+};
+
+export const resetUploadedFiles = () => {
+  return {
+    type: "uploaded",
+    filesUploaded: false,
+  };
+};
```

Comment:
Można użyć [redux toolkit](https://redux-toolkit.js.org/api/createAction)
Wtedy będzie to trochę wygodniejsze w obsłudze i czytelniejsze:

```js
const setUploadedFiels = createAction<bool>("uploaded")
```
I póżniej
```js
dispatch(setUploadedFiles(false)
```

Główną zaletą nad obecnym podejściem jest to, że nie musisz używać stringów.